app/graphql/types/query.rb

```diff
@@ -0,0 +1,11 @@
+module Types
+  class Query < GraphQL::Schema::Object
+    graphql_name 'Query'
+
+    field :model, Types::Model,
+      function: Queries::FetchField.new(type: Types::Model, model_class: ::Model)
+
+    field :models, Types::Model.connection_type,
```

Comment:
to fix the models query we need to apply connection logic manually here
```suggestion
    field :models, Types::Model.connection_type, connection: true,
```